ratpack-gradle/src/test/groovy/ratpack/gradle/functional/FunctionalSpec.groovy

```diff
@@ -135,7 +135,7 @@ abstract class FunctionalSpec extends Specification {
   }
 
   File getShadowJar() {
-    def f = file("build/libs/test-app-1.0-all.jar")
+    def f = file("build/libs/test-app-1.0.jar")
```

Comment:
I reverted this change because this is a bug in Shadow on Gradle 5.1+ due to this: https://docs.gradle.org/5.1/userguide/upgrading_version_5.html#change_in_behavior_for_tasks_extending_abstractarchivetask_or_subtypes_zip_jar_war_ear_tar

Instead, we should update the `build.gradle` in the test to explicitly set the `classifier` value.